src/components/PathwayCourse/index.js

```diff
@@ -466,20 +531,40 @@ function PathwayCourse() {
               </Grid>
             ))}
           </Grid>
-          {/* <Grid  sx={{mb:15}}align="center">
+          <Grid onClick={completedAll ? handleModal : handleSnackbar} sx={{mb:15}}align="center">
             <Grid sx={{mb:3}} >
               <img src={require("./asset/separator.svg")} alt="icon" />
             </Grid>
-            <Grid>
-              <img
-                src={require("./asset/certificate.svg")}
-                alt="certificate icon"
-              />
+            <Grid sx={{cursor: "pointer"}}>
+              {
+                completedAll ? 
+                <CertificateIconColored
+                  className={classes.certificateIcon}
+                /> : 
+                <CertificateIcon
+                  className={classes.certificateIcon}
+                />
+              }
               <Typography sx={{mt:2}} variant="body1" mb={2}>
-                {pathwayCourseData?.pathway} Certificate (Locked)
+                {pathwayCourseData?.pathway} Certificate
               </Typography>
             </Grid>
-          </Grid> */}
+            <Snackbar
+              open={openSnackbar}
+              autoHideDuration={6000}
```

Comment:
Should be referencing constants here for consistency in message display durations across the app or may be better to make this its own component to also ensure consistency of styles given in `ContentProps`.